docs/pages/component/props.mdx

```diff
@@ -63,7 +63,7 @@ Adjust the buffer settings. This prop takes an object with one or more of the pr
 | maxHeapAllocationPercent          | number | The percentage of available heap that the video can use to buffer, between 0 and 1                                                                                                              |
 | minBackBufferMemoryReservePercent | number | The percentage of available app memory at which during startup the back buffer will be disabled, between 0 and 1                                                                                |
 | minBufferMemoryReservePercent     | number | The percentage of available app memory to keep in reserve that prevents buffer from using it, between 0 and 1                                                                                   |
-
+| cacheSizeMB                       | number | Cache size in MB, it will allow applications to store video data for a while in the cache folder, it is useful to decrease bandwidth usage when repeating small videos. Android only.           |
```

Comment:
```suggestion
| cacheSizeMB                       | number | Cache size in MB, it will allow applications to store video data for a while in the cache folder, it is useful to decrease bandwidth usage when repeating small videos. Android only.           |

Note: cacheSizeMB is available only on Android and have following limitations:
- cache size should not be changed
- only 1 cache is used for all Video components, please do not pass the value to all Video component instanciated
```

Reply:
@lovegaoshi here is a proposal for doc update !